runtime/engine/src/test/java/io/aklivity/zilla/runtime/engine/test/EngineRule.java

```diff
@@ -213,15 +213,21 @@ public Statement apply(
             Configuration config = description.getAnnotation(Configuration.class);
             if (config != null)
             {
+                String resourceName;
                 if (configurationRoot != null)
                 {
-                    String resourceName = String.format("%s/%s", configurationRoot, config.value());
+                    resourceName = String.format("%s/%s", configurationRoot, config.value());
                     URL configURL = testClass.getClassLoader().getResource(resourceName);
+                    if (configURL == null)
+                    {
+                        configURL = new URL(testClass.getClassLoader().getResource(configurationRoot).toString() +
+                            "/" + config.value());
+                    }
```

Comment:
I already replied to this. Please find it here: https://github.com/aklivity/zilla/pull/141/files#r1068938356
If you have other suggestion, how to do this, please let me know.

Reply:
Sorry, the linked comment is not showing up, please summarize what breaks when omitted.

Reply:
This is needed for the 0-1 scenario, where we originally don't have the config file in place. This scenario should contain some configURL, if not, we can't watch the parent directory for changes. Since the current solution (testClass.getClassLoader().getResource(resourceName)) returns null if the file does not exists, we need to pass the configURL somehow. This is the workaround for that. Maybe there's a better solution, but this is the solution I could think of that requires the least amount of work.

Reply:
Yeah, I'd prefer we revert the changes in this code and find a different solution if possible, perhaps specific to the 0-1 test case.